src/code/z_quake.c

```diff
@@ -305,101 +387,121 @@ u32 Quake_RemoveFromIdx(s16 idx) {
     return false;
 }
 
-s16 Quake_Calc(Camera* camera, QuakeCamCalc* camData) {
-    f32 max;
-    f32 max2;
+s16 (*sQuakeCallbacks[])(QuakeRequest*, ShakeInfo*) = {
+    NULL,
+    Quake_CallbackType1,
+    Quake_CallbackType2,
+    Quake_CallbackType3,
+    Quake_CallbackType4,
+    Quake_CallbackType5,
+    Quake_CallbackType6,
```

Comment:
```suggestion
    NULL,                // QUAKE_TYPE_NONE
    Quake_CallbackType1, // QUAKE_TYPE_1
    Quake_CallbackType2, // QUAKE_TYPE_2
    Quake_CallbackType3, // QUAKE_TYPE_3
    Quake_CallbackType4, // QUAKE_TYPE_4
    Quake_CallbackType5, // QUAKE_TYPE_5
    Quake_CallbackType6, // QUAKE_TYPE_6
```
unless you're not a fan